Core/MIPS/IR/IRInterpreter.cpp

```diff
@@ -297,21 +317,36 @@ u32 IRInterpret(MIPSState *mips, const IRInst *inst, int count) {
 
 		case IROp::Vec4Pack32To8:
 		{
+#if defined(_M_SSE)
+			// Packs the upper bits, so we need to shift down. Then we can just use SSE packing.
+			__m128i val = _mm_srli_epi32(_mm_load_si128((const __m128i *)&mips->fi[inst->src1]), 24);
+			val = _mm_packs_epi16(_mm_packs_epi32(val, _mm_setzero_si128()), _mm_setzero_si128());
```

Comment:
`_mm_packus_epi16`?  Won't it saturate to 7F otherwise?

-[Unknown]

Reply:
Uh, yeah. I'm gonna go run the tests..

Reply:
Hopefully we have a test that actually catches that case...

-[Unknown]

Reply:
We have plenty of tests, and turns out the IRJit has a few bugs that need fixing, even before this change :) (in compares and float-to-int rounding, for example)..